whist/server/api/game/action.py

```diff
@@ -14,6 +14,18 @@
 router = APIRouter(prefix='/game')
 
 
+class PlayerNotReadyError(Exception):
+    """
+    Will check to see if player is ready before unreadying
+    """
+
+
+class GameNotFoundError(Exception):
+    """
+    Checks to see if user enters correct game_id
+    """
+
+
```

Comment:
The errors shouldn't be defined here. `GameNotFoundError` is already defined somewhere else. See above.